tools/uebuild/uebuild.nim

```diff
@@ -472,13 +480,16 @@ when isMainModule:
           mode = key
         of otPlatform:
           platform = key
+        of otCpu:
+          cpu = key
         of otEngineLocation:
           engineDir = key
         expectedOptType = otCommand
       of cmdShortOption:
         case key:
           of "d": expectedOptType = otProjectDir
           of "p": expectedOptType = otPlatform
+          of "c": expectedOptType = otCpu
```

Comment:
no need for shortcut here, shortcuts should only be there for the most common options.